### PR TITLE
39 feat news page viewmodel

### PR DESCRIPTION
**Pull request by ClementJosso**
Created the model and viewmodel, still needs fixing when fetching from the database.

### CODE REVIEW

File: lib/viewmodels/news.dart [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+          .from('news')
+          .select()
+          .eq('user', supabase.auth.currentUser!.id)
+          .eq('date', dateString)
+          .single();
+
+      // If there is no news for this day AND date == today, call the transcript function
+      // If it's the right day but before the time, tell them to come back. If it's the right day after the time, say there is a problem.
+
+      List<String> texts = List<String>.from(jsonDecode(res['texts']));
```
Comment (quartztz) [inline lib/viewmodels/news.dart:25]: Might be refactored at some point but for now it works!

**Comment by ClementJosso** [comment]
At the moment the fetching works. Next step is to implement the edge function to write in the DB.

**Review by quartztz — DISMISSED** [review]
Code looks great ! Tested on Android and works fine according to specs. LGTM!

File: lib/main.dart [hunk 2]
```diff
@@ -36,23 +42,18 @@ class _AppState extends State<App> {
...
-          colorScheme: ColorScheme.fromSeed(seedColor: Colors.deepPurple),
-          useMaterial3: true,
-        ),
-        home: Scaffold(
-          appBar: AppBar(
-            backgroundColor: Theme.of(context).colorScheme.inversePrimary,
-            title: const Text('ActualIA'),
-          ),
-          body: body,
-        ));
```
Comment (Thechi2000) [inline lib/main.dart:56]: Modifying this breaks the Auth page:
image

**Review by Thechi2000 — CHANGES_REQUESTED** [review]
The Auth page theme is broken.

**Review by quartztz — APPROVED** [review]
missed that one :D good!

**Review by Thechi2000 — APPROVED** [review]

**Pull request merged**